test/system/bans_test.rb

```diff
@@ -119,6 +134,7 @@ def test_banned_user_can_see_the_ban_notification_for_longest_repetitive_ban
 
     visit auctions_path
     assert(page.has_css?('div.ban', text: text))
+    assert_not(page.has_css?('div.ban > p'))
```

Comment:
`has_no_css?`

Reply:
Other tests here uses assert_not method, so I prefer to keep the codestyle constant.

Reply:
`has_no_css?` is much more readable and more simple, than `assert_not(page.has_css?)`.

Reply:
Well, here we would like to check what given block doesn't shown to user at all, so check for text maybe wouldn't the best idea, but for absence of css is.
Changed to has_no_css? .